render.js

```diff
@@ -0,0 +1,17 @@
+const Renderer = function () {
+    const renderPosts = function (posts) {
+        $('#posts').empty()
+        for (post of posts) {
+            let innerPost = `<div class='post' data-id='${post.id}'><p class="post-text">${post.text}</p>`
+            for (let comment of post.comments) {
+                innerPost += `<p class ='comments' data-id='${comment.id}'><span class="delete-comment">x </span>${comment.text}</p>`
+            }
+            innerPost += '<p><input placeholder="Got something to say?"></input><button class="addComment">Comment</button></p> <button class="delete">Delete post</button></div>'
+            $('#posts').append(innerPost)
+        }
+    }
+    return renderPosts
```

Comment:
You should probably return an object. It's better practice. If you needed to add to this application down the road, then you would have to change all the places where you invoked this in `main`. If it's an object you can just add methods to it and everything in main will just work the same.